app/src/main/java/atlant/moviesapp/activity/SeasonsActivity.java

```diff
@@ -126,6 +145,12 @@ public void onLongClick(View view, int position) {
 
     }
 
+    public boolean isNetworkAvailable() {
```

Comment:
Please, include the isNetworkAvailable() method in a Base class (either a BaseActivity class or a BaseFragment class depending on where exactly it is used), as to avoid duplication of the code.
You have isNetworkAvailable an exact same method declared many many times through the whole android apps components.